t/ui/27-plugin_obs_rsync_obs_status.t

```diff
@@ -153,5 +232,35 @@ $t->get_ok('/admin/obs_rsync/queue')->status_is(200, 'jobs list')->content_like(
 $t->get_ok('/admin/obs_rsync/')->status_is(200, 'project list')->content_like(qr/published/)->content_like(qr/dirty/)
   ->content_like(qr/publishing/);
 
+subtest 'build service ssh authentication' => sub {
+    is($helper->is_status_dirty('ProjWithAuth'), 1, 're-authenticate with ssh auth');
+};
+
+subtest 'build service authentication: signature generation' => sub {
+    $mocked_time = 1664187470;
+    note "time right now: " . time();
+
+    is(time(), $mocked_time, 'Time is not frozen!');
+
+    is($helper->is_status_dirty('ProjTestingSignature'), 1, 'signature matches fixture');
+    $mocked_time = undef;
```

Comment:
I guess our preferred style would be to avoid brackets and to use single-quotes when not using interpolation and to not add too many blank lines:
```suggestion
    $mocked_time = 1664187470;
    note 'time right now: ' . time();
    is time(), $mocked_time, 'time is mocked';
    ok $helper->is_status_dirty('ProjTestingSignature'), 'signature matches fixture';
    $mocked_time = undef;
```

Reply:
This should be a linting rule.

Reply:
yes, it should. Can you make that a rule? Because I wouldn't know how

Reply:
Good news: it already exists

https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitInterpolationOfLiterals

I'll add it to `os-autoinst-common` and slowly bring it here.